app/assets/javascripts/initializers/initializeOnboardingTaskCard.js.erb

```diff
@@ -8,7 +8,13 @@ function initializeOnboardingTaskCard() {
 
   var taskCard = document.getElementsByClassName("onboarding-task-card")[0];
 
-  if (taskCard) {
+  if (taskCard == null) { return }
+
+  var createdAt = new Date(userData().created_at);
+  var now = new Date();
+  var aWeekAgo = now.setDate(now.getDate() - 7);
+
+  if (createdAt > aWeekAgo) {
```

Comment:
We should include something in the documentation for this function that explains the "one week ago" logic and changing the style on the card accordingly.

Reply:
Yeah I actually updated this before you reviewed https://github.com/thepracticaldev/dev.to/pull/7369/commits/236906112fc00379657932befe38b3bd6e485af2 🤔

Reply:
weird. i blame github!!